test-harness/modules/generic_test_code.py

```diff
@@ -8,7 +7,52 @@
 log = logging.getLogger(__name__)
 
 
-def generic_unknown_user_is_forbidden_test(ar, auth_header, path):
+def ping_mesos_agent(ar,
```

Comment:
This is because GH folds files which produce too much diff output. Try expanding `test-harness/tests/test_cache.py ` and you will find a lot of uses of this function.